challenge_5/python/ekstrah/src/FindTheDifference.py

```diff
@@ -0,0 +1,8 @@
+class Solution(object):
+    def findTheDifference(self, s, t):
```

Comment:
The [Counter](https://docs.python.org/3/library/collections.html#collections.Counter) class can serve you well here also.
If you would prefer to stay away from it for some reason you could implement your own counter a dict with the frequency of each letter for example. 

To avoid the identical string output problem you can either test to see if the strings are the same or return a default value in the case you cannot find an extra letter.